frontend/src/components/pages/CreateReview/CreateReviewPage.tsx

```diff
@@ -0,0 +1,276 @@
+import { AddIcon, ArrowBackIcon, DeleteIcon } from "@chakra-ui/icons";
+import {
+  Box,
+  Button,
+  ButtonGroup,
+  Heading,
+  IconButton,
+  Input,
+  Radio,
+  RadioGroup,
+  Stack,
+  Text,
+} from "@chakra-ui/react";
+import React, { useEffect, useState } from "react";
+
+import { Book } from "../../../types/BookTypes";
+// import { Option } from "../../../types/TagTypes";
+// import BookModal from "./BookModal";
+import DeleteModal from "./DeleteBookModal";
+import DeleteReviewModal from "./DeleteReviewModal";
+import PublishModal from "./PublishModal";
+import ReviewEditor from "./ReviewEditor";
+import SingleBook from "./SingleBook";
+
+const CreateReview = (): React.ReactElement => {
+  // const [tagsSelected, setTagsSelected] = useState<Option[]>([]);
+  // const [showBookModal, setShowBookModal] = useState<boolean>(false);
+  const [showDeleteBookModal, setShowDeleteBookModal] = useState<boolean>(
+    false,
+  );
+  const [showPublishModal, setShowPublishModal] = useState<boolean>(false);
+  const [showDeleteReviewModal, setShowDeleteReviewModal] = useState<boolean>(
+    false,
+  );
+  const [deleteBookIndex, setDeleteBookIndex] = useState<number>(-1);
+  const [books, setBooks] = useState<Book[]>([]);
+  const [review, setReview] = useState("");
+  const [featured, setFeatured] = useState("0");
+
+  // const onBookModalClose = () => setShowBookModal(false);
+  const onDeleteBookModalClose = () => setShowDeleteBookModal(false);
+  const onPublishModalClose = () => setShowPublishModal(false);
+  const onDeleteReviewModalClose = () => setShowDeleteReviewModal(false);
+
+  // const handleTagSelected = (e: Option[]) => {
+  //   setTagsSelected(e);
+  // };
+
+  // adds a book to the list of books
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
can you get rid of this es-lint disable? i think all the variables are used here